docs/pages/divider.js

```diff
@@ -1,64 +1,309 @@
 // @flow strict
 import type { Node } from 'react';
-import PropTable from '../components/PropTable.js';
 import PageHeader from '../components/PageHeader.js';
-import CardPage from '../components/CardPage.js';
 import MainSection from '../components/MainSection.js';
+import Page from '../components/Page.js';
 
-const cards: Array<Node> = [];
-const card = (c) => cards.push(c);
+export default function DividerPage(): Node {
+  return (
+    <Page title="Divider">
+      <PageHeader
+        name="Divider"
+        description="If you have two things that need to be separated, put a `Divider` between them."
+        defaultCode={`
+function Example() {
+  const Block = ({ title, text }) => (
+    <Flex direction="column" gap={2}>
+      <Heading size="sm">{title}</Heading>
+      <Text size="md">{text}</Text>
+    </Flex>
+  );
 
-card(
-  <PageHeader
-    name="Divider"
-    description="If you have two things that need to be separated, put a `Divider` between them."
-  />,
-);
+  return (
+    <Flex>
+      <Box paddingX={4}>
+        <Block title="Discover ideas" text="Use the searh bar to discover ideas, people and trends. Explore suggested topics or search for topics of your own." />
+      </Box>
+      <Divider />
+      <Box paddingX={4}>
+        <Block title="Create Pins" text="Upload an image from your computer or mobile device to create a Pin. You can also create Pins from images yu find online." />
+      </Box>
+    </Flex>
+  );
+}
+`}
+      />
+      <MainSection name="Usage guidelines">
+        <MainSection.Subsection>
+          <MainSection.Card
+            description={`
+Dividers can be a subtle way to separate content, however, white space can be used to accomplish the same goal. Use dividers sparingly to avoid creating unnecessary clutter. Dividers should be used mainly to separate groups of related content.
+        `}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Best practices">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Use whitespace primarily to separate groups of related content"
+            defaultCode={`
+function Example() {
+  const Block = ({ title, text }) => (
+    <Flex direction="column" gap={2}>
+      <Heading size="sm">{title}</Heading>
+      <Text size="md">{text}</Text>
+    </Flex>
+  );
 
-card(<PropTable props={[]} />);
+  return (
+    <Flex direction="column" gap={10}>
```

Comment:
Can we mix it up a bit with these examples? Seeing the same copy so many times in a row is a bit boring and doesn't engage the reader.